apps/src/lib/util/StatsigReporter.js

```diff
@@ -7,6 +8,7 @@ import {
   getEnvironment,
   isProductionEnvironment,
   isDevelopmentEnvironment,
+  createUuid,
```

Comment:
Nit: What does the second "u" stand for? Should it be "createUid"?

Reply:
UUID stands for "Universally Unique Identifier".

Reply:
Ah thank you!